### PR TITLE
[AK,MC] Revisit springer-media documentation to meet latest guidelines

### PR DESCRIPTION
**Pull request by morgaan**
- Revisit README.md to meet latest guideline    
- Enhance demo with more examples

### Enhanced demo page

screenshot-of-springer-media-full-page-demo

### CODE REVIEW

File: toolkits/springer/packages/springer-media/README.md [hunk 1]
```diff
@@ -1,42 +1,79 @@
...
-            </h4>
-            <p>Text body</p>
-        </div>
-    </li>
-</ul>
+# Springer Media
+
+[![NPM version][badge-npm]][info-npm]
+
+The Springer Media component is derived on the media object pattern we see all over the web. [Named by Nicole Sullivan](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/)
```
Comment (jon-stevens) [inline toolkits/springer/packages/springer-media/README.md:5]:
```suggestion
The Springer Media component is derived from the media object pattern we see all over the web. [Named by Nicole Sullivan](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) 
```

File: toolkits/springer/packages/springer-media/README.md [hunk 2]
```diff
@@ -1,42 +1,79 @@
...
-            <p>Text body</p>
-        </div>
-    </li>
-</ul>
+# Springer Media
+
+[![NPM version][badge-npm]][info-npm]
+
+The Springer Media component is derived on the media object pattern we see all over the web. [Named by Nicole Sullivan](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) 
+it refers to a two-column box with an image on one side and descriptive text on the other.
```
Comment (jon-stevens) [inline toolkits/springer/packages/springer-media/README.md:6]:
```suggestion
. It describes a two-column box with an image on one side and descriptive content on the other.
```

File: toolkits/springer/packages/springer-media/README.md [hunk 3]
```diff
@@ -1,42 +1,79 @@
...
+
+## Installation
+
+To use the Media component, enter the following command in your Terminal:
+
+```
+npm install @springernature/springer-media
+```
+
+Import the installed component code in your `scss` file:
```
Comment (jon-stevens) [inline toolkits/springer/packages/springer-media/README.md:26]:
```suggestion
Import the Sass:
```

File: toolkits/springer/packages/springer-media/README.md [hunk 4]
```diff
@@ -1,42 +1,79 @@
...
+@import '@springernature/springer-media/scss/10-settings/media';
+@import '@springernature/springer-media/scss/50-components/media';
+
+@import '@springernature/brand-context/default/scss/60-utilities/hiding.scss';
+```
+## How it works
+### Stacked
+
+It is possible to have the figure and text shown as one column. In order to do that add 
+`c-media--stagged` to the element carrying the `c-media` class.
```
Comment (jon-stevens) [inline toolkits/springer/packages/springer-media/README.md:38]: Should this be `c-media-stacked`?

File: toolkits/springer/packages/springer-media/README.md [hunk 5]
```diff
@@ -1,42 +1,79 @@
...
+the`c-media__body` class.
+
+When the component is stacked, left figure is shown above the text and right figure is shown below.
+
+To display the **text only**, omit the element carrying the `c-media__figure` class.
+
+### Play icon
+
+The component provides an option to show a play icon (triangle) on the figure, which can be used to link to video material.
+In order to do that markup the figure as shown in the code snippet below:
```
Comment (jon-stevens) [inline toolkits/springer/packages/springer-media/README.md:57]:
```suggestion
In order to do that, mark up the figure as shown in the code snippet below:
```

File: toolkits/springer/packages/springer-media/README.md (hunk 4 above)
Comment (andreaskoscinski) [reply toolkits/springer/packages/springer-media/README.md:38]: Guess you're right

File: toolkits/springer/packages/springer-media/view/media.hbs [hunk 6]
```diff
@@ -1,16 +1,38 @@
...
+					</div>
+				</div>
+			{{else}}
+				<img src="{{src}}" class="c-media__figure" alt="{{alt}}">
+			{{/if}}
+		{{/with}}
 	{{/with}}
 	<div class="c-media__body">
-		<h4 class="c-media__title" aria-level="{{level}}">
+		<h{{level}} class="c-media__title">
```
Comment (jon-stevens) [inline toolkits/springer/packages/springer-media/view/media.hbs:17]:
```suggestion
		<h4{{level}} class="c-media__title">
```

**Review by jon-stevens — APPROVED** [review]

**Pull request merged**